frontend/web/js/tests.js

```diff
@@ -52,21 +52,17 @@ function select_extension() {
 }
 
 
-// CUSTOM VARIANT SELECT
 function select_variant() {
-    const my_variant = [],
-        target_num = 2;
-
-    
-    let variant1 = document.getElementById('variant1');
-    let variant2 = document.getElementById('variant2');
-    if(variant1){
-        return variant1.value
-    }else{
-        return variant2.value
-    }
+    const target_num = 2;
 
-    
+    let variant = null;
```

Comment:
No need to create a null variable before the loop. Create the variable on-demand inside the loop